src/components/modals/TransactionAdvanceDetails.js

```diff
@@ -1,13 +1,11 @@
 // @flow
 
-import type { EdgeDenomination, EdgeTransaction } from 'edge-core-js'
```

Comment:
Squash this commit with the previous one

Reply:
'Updated the history. There should only be two commits now.